website/cloudbuild.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: 'gcr.io/cloud-builders/npm'
+    args: ['ci']
+  - name: 'gcr.io/cloud-builders/npm'
+    args: ['run', 'lint-md']
+  - name: 'gcr.io/cloud-builders/bazel'
+    args: ['run', '//runsc', '--', 'help', 'syscalls', '-o', 'json', '--filename', '/workspace/website/syscalls.json']
+  - name: 'gcr.io/cloud-builders/docker'
```

Comment:
We may want to think about compressing this all down into gvisor-website and have one build step here that just uses that docker image and runs 'make'. A little tricky since the image would need to have bazel, jekyll, npm etc. all in one image but complexity would be contained there and would simplify other parts of the project. It would also remove duplication between here and the Makefile.

Reply:
There is now just one build step.